portugues/pescaria/no_mar_da_bahia/no_barco.md

```diff
@@ -0,0 +1,7 @@
+[Lembro que quando estava fazendo um Nanodegree na Udacity](https://github.com/albertoafaj/create-your-own-adventure/blob/master/portugues/udacity/udacity.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.